identifiers/country-ca/README.md

```diff
@@ -0,0 +1,11 @@
+## Federal Electoral Districts
```

Comment:
It might be nice to add to this a list of files which are generated by scripts vs manually maintained